client/src/components/cluster/RecommendedCategories/RecommendedCategories.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import CategoryContainer from './CategoryContainer.js';
+import {listOne, listTwo} from './mockData.js';
+
+class RecommendedCategories extends Component {
+    render() {
+        const makeRow = item => (
+            <CategoryContainer
+            category={item.category}
+            url={item.url}
+            />
+        );
+    }
+}
```

Comment:
This doesnt render anything. Did you forget to render the CategoryContainer?

Reply:
I added RecommendedCategories to the App.js file. Does that resolve the problem?

Reply:
Lemme rephrase. If you look at your CategoryContainer component, you'll see it returns something in the render function. Nothing is being returned in the RecommendedCategories's render function